COMMITTERS.adoc

```diff
@@ -0,0 +1,173 @@
+====
+GitHub won't detect issues properly if you do something like
+`Fix #444 #555`.
+====
+
+== Affecting Labels and Milestones
+
+Affecting labels and milestones is very important in our process.
+
+Before each release, we check that we don't have any orphan i.e.
+closed issues that don't have either a milestone affected or
+some of the "excluding" labels.
+
+Thus:
+
+* when you merge a pull request, please affect it with the current milestone marked with "master".
+* if the pull request has issues associated (i.e. if the pull request fixes some issues),
+  you should also affect the milestone assigned to the issues.
+* if you close a pull request because the committers have decided to not merge it,
+  please add the appropriate `triage/` label: `triage/invalid`, `triage/out-of-date`,
+  `triage/wontfix` are usually in order.
+* if you close an issue because it has been fixed, either add the milestone
+  if you can find it easily or use the `triage/out-of-date` if you can't.
+* if you close an issue for any other reason, one of the aforementioned `triage/` labels
+  is probably adequate.
+
+== Backporting Process
+
+When we release a new version of Quarkus, we usually do a bugfix
+release a couple of weeks after.
+
+Everytime, we do a release, we create a release branch to host
+the commits for these bugfix releases.
+
+All the pull requests are merged in master so for the new feature 
+release of Quarkus.
+They won't be integrated in the previous version branch.
+
+One of your pull requests might qualify for being backported to this
+bugfix branch.
+
+Good examples of that might be:
+
+* bugfixes
+* documentation fixes
+* usability fixes
+
+Obviously, the barrier is higher for large pull requests as
+they might be more risky to backport.
+But sometimes, we just have to backport them anyway.
+
+If you think your pull request or the pull request you review might be a good backport candidate,
+please add the `triage/backport?` label.
+
+The question mark is important:
+it is not automatic and we carefully review each pull request before backporting.
+
+Thus, if not entirely clear, don't hesitate to add a comment to the pull request
+when adding the label.
+
+And don't be surprised if we come to you with some questions about it
+when we prepare the next bugfix release.
+
+== Good First Issues
+
+We need to find the right balance between fixing the issue right away
+or try to get a new contributor onboard.
+
+It's not always easy to find it but if you think the issue is appropriate,
+affecting it the `good first issue` label for some time might be a good thing.
+
+Obviously, critical bugs are not good candidates :).
+
+== I Did Something Wrong, What Should I Do?
+
+Take a deep breath and don't worry, it happens.
+
+Just ping https://github.com/gsmet/[Guillaume Smet (@gsmet)] and https://github.com/geoand/[Georgios Andrianakis (@geoand)]
+either in the pull request or on https://quarkusio.zulipchat.com/[our Zulip] and we will find a solution.
```

Comment:
:+1:

Reply:
how about we create a `@committerhelp` (?) group alias to use here instead of specific individuals ? like we done for devtools, and others.

Reply:
We could have an alias. TBH, my fear was that people could get worried of looking dumb or something like that. Thus limiting the number of persons would help. You know your mistake won't be seen by 20 persons.

But I can live with an alias if others think I'm overthinking it.

Reply:
+1 on a group

Reply:
I don't mind either way. It's going to be a very limited group one way or another

Reply:
@quarkusio/committerhelp now exist both on github and zulip; members being gsmet and geoand at this time.